plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -32,9 +32,17 @@
       choices: [ 'absent', 'present' ]
       default: 'present'
       type: str
+    apply_live:
+      description:
+      - Stage packages and overlay them on the running system tree.
+      - Corresponds to the C(--apply-live) option. Default behavior (C(no)); C(yes) overlays the requested package(s) to the live system.
+      required: false
+      default: "no"
+      type: bool
```

Comment:
```suggestion
      type: bool
      version_added: '4.2.0'
```

@Andersson007 Could you please let us know this value?

Reply:
@Akasurde which value do you mean? If it's the value of version_added, as i can see it should be 4.2.0 but I think @felixfontein is the right person to ask:)

Reply:
Yes, I am referring to `version_added`. I was not sure so I thought it is better to ask.

Reply:
The next minor release is 4.2.0, so that value is correct.